### ISSUE TITLE
Does nothing for me (Xbox Game Pass)

### ISSUE THREAD

**Issue by ap3nr0ts** [Lyall/P3RFix#3]
I've downloaded your zip, put it in the folders you said, and changed this in the .ini:

```
[Custom Resolution]
; Enter your chosen resolution.
; You can leave these at 0 if you want your desktop resolution used instead.
Enabled = true
Width = 1280
Height = 800
```

The game stays at the settings I've last used (2560x1560 fullscreen, which I'm using to stream to my Steam Deck right now). I'd rather use 1280x800 if possible. Thanks for any tips or fixes.

Edit: it works windowed, but I see you can't adjust for the interface yet (bars above and below menu animations). For now, I'll stick with 16:9 then, but maybe others have the same problem with it only enabling via windowed.

**Comment by Lyall** [comment]
It overwrites the 800x450 resolution option in-game with your custom resolution entry. I should have mentioned that in the config  file. Have you checked the resolution list in-game? It should be the first entry. Also bear in mind that UE4 games that run borderless run at the desktop resolution so the custom resolution will only work in fullscreen or windowed.

**Comment by ap3nr0ts** [comment]
> It overwrites the 800x450 resolution option in-game with your custom resolution entry. I should have mentioned that in the config file. Have you checked the resolution list in-game? It should be the first entry. Also bear in mind that UE4 games that run borderless run at the desktop resolution so the custom resolution will only work in fullscreen or windowed.

Yeah, I got that resolution in windowed, but it disappears when going fullscreen again (only shows 850x400 again). Don't know why. The bars above and below the menu animations are probably hard to fix? I thought they wouldn't appear, since you can also have 4:3 resolutions, scaling-wise.

**Comment by Lyall** [comment]
> Yeah, I got that resolution in windowed, but it disappears when going fullscreen again (only shows 850x400 again). Don't know why. The bars above and below the menu animations are probably hard to fix? I thought they wouldn't appear, since you can also have 4:3 resolutions, scaling-wise.

Aha, I'm seeing that now. You're totally right. Looks like fullscreen uses a generated resolution list instead of the fixed one for windowed. I think I'll probably ditch the adding resolution approach and just override it in the next version. That way, whatever you have set in the config file is applied straight away in windowed/fullscreen.

> Edit: with bars I mean just clear backgrounds, without any menu animations/overlay.

You mean the gaps like shown in this screenshot at the bottom?

![P3R_2024_02_03_00_14_32_600](https://github.com/Lyall/P3RFix/assets/695941/598ee40d-e854-4f9a-874a-02ef85beefa8)

If so, then I can partially address them. I forgot to add a HUD option for <16:9 aspect ratios so it would look like this with them applied. Should be added in the next release.

![P3R_2024_02_03_00_14_24_389](https://github.com/Lyall/P3RFix/assets/695941/1293125a-dd30-409c-9cce-9756e95d6a1a)

**Comment by ap3nr0ts** [comment]
@Lyall Thanks! Those are exactly my issues! Is it also possible to stretch te content? For instance, you also have a full UI with 4:3 content. So with 16:10, it could maybe just be another way to cut it all off. Think that would look even better, but may be too hard of get cut off from the sides again.

**Comment by Lyall** [comment]
Both these issues should be addressed in the latest (v1.0.1) release. If they work for you feel free to close the issue and if not, let me know.

**Comment by ap3nr0ts** [comment]
You are awesome. I think everything looks great now.
